content/blog/nielx/2022-12-27_http_network_services_preview_in_r1_beta_4.md

```diff
@@ -0,0 +1,111 @@
++++
+type = "blog"
+title = "HTTP Network Services Preview in R1 Beta 4"
+author = "nielx"
+date = "2022-12-27 00:00:01+00:00"
+tags = ["haiku", "software", "development", "netservices"]
++++
+
+The newest beta of Haiku will have a preview of a redesigned, modern HTTP library as the initial part of an renewed Network Services Kit. The primary goal of including the library is to get developers to experiment with it and give feedback on how it works within their software. The secondary goal is to gather some feedback on the use of modern C++ and some additional experimental features. This article provides a background to the new kit, some pointers on how to get started, and some notes on experimental designs that utilize modern C++ features.
```

Comment:
now that beta4 is out, this "will have" is obsolete

Reply:
Updated the intro paragraph (and bumped the publishing date)